sktime/annotation/clasp.py

```diff
@@ -258,22 +258,6 @@ def _predict(self, X):
         Y : pd.Series or an IntervalSeries
             Change points in sequence X.
         """
-        change_points = self._predict_points(X)
-        return change_points
-
-    def _predict_points(self, X):
```

Comment:
Removed overriden `_predict_points` method based on understanding that child detectors should only override the `_predict` method. `predict_points`, `predict_segments` and `transform` should be provided by the base class.

There should be no regression because the `_predict` method in ClaSP was simply calling the overridden `_predict_points` method. Now that the logic from `_predict_points` has been moved directly into `_predict`, the behavior remains unchanged. Additionally, in [base.py](https://github.com/sktime/sktime/blob/main/sktime/detection/base/_base.py#L597-L610), the base `_predict_points` method calls `_predict`, ensuring it continues to perform exactly as before, with the overridden `_predict_points` removed.